bpfd-operator/config/bpfd-deployment/daemonset-csi.yaml

```diff
@@ -27,6 +27,10 @@ spec:
       - key: node-role.kubernetes.io/master
         operator: Exists
         effect: NoSchedule
+      # Specify FS group so that the agent can use the bpfd socket
+      # without needing to be root.
+      securityContext:
+        fsGroup: 2000
```

Comment:
I read https://kubernetes.io/docs/tasks/configure-pod-container/security-context/ on the FS Group, but it just gave an example of 2000, didn't explain where 2000 came from. Is that just standard practice to use 2000?

Reply:
Good question I used it completely arbitrarily, do you have something else you'd rather have it be?

Reply:
I am not familiar with it, so I was more asking.